e2e-tests/pom.xml

```diff
@@ -32,6 +32,11 @@
                 <scope>test</scope>
                 <type>test-jar</type>
             </dependency>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-validation</artifactId>
+                <version>2.5.0</version>
+            </dependency>
```

Comment:
this is not the main root pom file. furthermore, the version should be `${springboot.version}` as all other dependencies

Reply:
Done